ads/insticator.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function insticator(global, data) {
+  // validate passed data attributes
+  validateData(data, ['siteId', 'embedId']);
+
+  // create insticator markup
+  global.document.getElementById('c').appendChild(createTemplate(data.embedId));
+
+  // create ads and embed
+  createAdsAndEmbed(data.siteId, data.embedId);
+
+  // envoke AMP library
+  loadScript(global);
```

Comment:
I think this is not doing anything?

Reply:
Removed. Thought this was necessary from the AMP side.